<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云直播服务器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/new/prefix1/logo_R.ico</normaloff>:/new/prefix1/logo_R.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>在线数：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>房间数：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>客户端：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLCDNumber" name="LCD_user"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="LCD_room"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="LCD_socket"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>直播公告：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_public"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_send_public">
          <property name="text">
           <string>发送给所有用户</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_listen">
            <property name="text">
             <string>listen</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_reboot">
            <property name="text">
             <string>reboot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>close</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>服务器日志：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>服务器压力：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="te_debug"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
